splitio/api/commons.py

```diff
@@ -125,5 +125,5 @@ def build_fetch(change_number, fetch_options, metadata):
     if fetch_options.change_number is not None:
         query['till'] = fetch_options.change_number
     if fetch_options.sets is not None:
-        query['sets'] = ','.join(fetch_options.sets)
+        query['sets'] = ','.join(sorted(fetch_options.sets))
```

Comment:
correct me if I'm wrong but this seems to be executed every single time that the logic makes a fetch, if that so, this is not as performant as doing this when you set/create the FetchOptions

Reply:
The FetchOptions object is created every time a sync is performed in sync.segment synchronize_segment() and sync.splits synchronize_splits(), so it would not make a difference to move the line there.
Unless we create the FetchOoptions object in the sync initializers, however, we still need to update the query sets for any changes, which also means looping through the sets.
Let me know how you want to proceed.

Reply:
Yeah, we still need to update the sets for any possible changes, in the sync class before calling the fetch api, which means looping through the sets, will update the PR

Reply:
PR is updated